docs/overview.md

```diff
@@ -13,22 +16,49 @@ As a versatile solution, ScalarDB supports a range of databases, including:
 
 For details on which databases ScalarDB supports, refer to [Supported Databases](scalardb-supported-databases.md).
 
+## Why ScalarDB?
+
+There are several solutions, such as global transaction managers, data federation engines, and HTAP systems, built towards similar goals, but they are limited in the following perspectives:
```

Comment:
The suggestion looks like something is missing.
```exist that are built toward``` should be ```, built toward``` or ```, which built toward``` or something?
Can you check it again?

Reply:
I can see how it might be awkward to read. How about one of the following instead?:

```suggestion
Several solutions, such as global transaction managers, data federation engines, and HTAP systems, are built with similar goals, but they are limited in the following perspectives:
```

```suggestion
Several solutions, such as global transaction managers, data federation engines, HTAP systems, have similar goals, but they are limited in the following perspectives:
```

Reply:
@josh-wong Thanks! I applied the latter.